wrappers/python/simtk/openmm/app/topology.py

```diff
@@ -404,3 +436,32 @@ def __init__(self, name, element, index, residue, id):
 
     def __repr__(self):
         return "<Atom %d (%s) of chain %d residue %d (%s)>" % (self.index, self.name, self.residue.chain.index, self.residue.index, self.residue.name)
+
+class Bond(tuple):
```

Comment:
What about making it a namedtuple subclass so you can do attribute access on it?